fe/fe-core/src/main/java/com/starrocks/sql/optimizer/transformer/RelationTransformer.java

```diff
@@ -415,7 +415,7 @@ public LogicalPlan visitCTE(CTERelation node, ExpressionMapping context) {
 
     @Override
     public LogicalPlan visitSubquery(SubqueryRelation node, ExpressionMapping context) {
-        LogicalPlan logicalPlan = visit(node.getQuery());
+        LogicalPlan logicalPlan = transform(node.getQuery());
```

Comment:
we should use transform instead of visit for relation which may use cte ?

Reply:
yes